src/NHSD.BuyingCatalogue.Solutions.API/Controllers/EpicsController.cs

```diff
@@ -31,5 +36,21 @@ public ActionResult GetAll([FromRoute][Required]string id)
 
             return Ok(result);
         }
+
+       // <summary>
+       // Updates the solution's epics.
+       // </summary>
+       // <param name="id"></param>
+       // <param name="viewModel"></param>
+       // <returns>A Task representing the operation to update the details of the epics section</returns>
+      [HttpPut]
+      [Route("{id}/sections/epics")]
+      [ProducesResponseType(StatusCodes.Status400BadRequest)]
+      [ProducesResponseType(StatusCodes.Status204NoContent)]
+      [ProducesResponseType(StatusCodes.Status404NotFound)]
+      public async Task<ActionResult> Update([FromRoute] [Required] string id,
+          [FromBody] [Required] UpdateEpicsViewModel viewModel) =>
+          (await _mediator.Send(new UpdateEpicsCommand(id, new HashSet<IClaimedEpic>(viewModel?.ClaimedEpics)))
```

Comment:
Could we rename the class UpdateEpicsCommand to UpdateClaimedEpicsCommand & follow that pattern for the handler etc..?